crates/bevy_math/src/bounding/bounded3d/primitive_impls.rs

```diff
@@ -303,14 +303,32 @@ impl Bounded3d for Torus {
     }
 }
 
+impl Bounded3d for Ramp {
+    fn aabb_3d(&self, translation: Vec3, rotation: Quat) -> Aabb3d {
+        let points = [
```

Comment:
I think this line is mostly duplicate with the gizmos one, could we share the lists and also document what points they are supposed to represent (I assume each corner of the wedge, but that might be unclear to others reading the code)

Reply:
I'm not sure that's a good idea since they are in different crates